vega_sim/wallet/slim_wallet.py

```diff
@@ -17,6 +18,9 @@
 
 logger = getLogger(__name__)
 
+dotenv.load_dotenv()
+DEFAULT_KEY_NAME = os.environ.get("DEFAULT_KEY_NAME")
```

Comment:
Can we move this read inside the `__init__` of the wallets? e.g. have a `DEFAULT_KEY_NAME = 'foo'` out here then inside `self.default_key = os.environ.get("DEFAULT_KEY_NAME", DEFAULT_KEY_NAME)`. I like to keep as much as possible out of running at import time (plus reading at class creation time lets you vary it if you want at a later time point)

Reply:
Let's also rename to something more Vega specific (VEGA_DEFAULT_KEY_NAME would work)

Reply:
No that sounds good, I like that a lot more